cross/bash/Makefile

```diff
@@ -5,7 +5,7 @@ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = http://ftp.gnu.org/gnu/bash
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS =
+DEPENDS = cross/ncurses
```

Comment:
i added it because the repo uses ncures-6 and on the synology (5.2) we only have 5.9 installed. so i got an error that bash cannot load libncurses.6